evm/p2p/kademlia.py

```diff
@@ -8,19 +8,30 @@
 import time
 import urllib
 from functools import total_ordering
+from typing import AnyStr, Callable, Dict, List, Set, Sized, Tuple  # noqa: F401
 
 from eth_utils import (
     decode_hex,
     encode_hex,
     force_bytes,
 )
 
-from eth_keys import keys
+from eth_keys import (
+    datatypes,
+    keys,
+)
 
 from evm.utils.keccak import keccak
 from evm.utils.numeric import big_endian_to_int
 
 
+# Workaround for import cycles caused by type annotations:
+# http://mypy.readthedocs.io/en/latest/common_issues.html#import-cycles
+MYPY = False
```

Comment:
Can we use [`typing.TYPE_CHECKING`](https://docs.python.org/3.5/library/typing.html) here instead?

Reply:
Oh, yeah, I had to use this in eth-keys because apparently there's no TYPE_CHECKING in py3.4, but here it should be ok. Will change